src/pages/case-study/doctorlink-setting-up-a-design-system.js

```diff
@@ -0,0 +1,689 @@
+import React from 'react'
+import { graphql } from 'gatsby'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+
+import { Grid, Row } from '../../components/grid'
+import BlueBackground from '../../components/Common/BlueBackground'
+import GreyBackground from '../../components/Common/GreyBackground'
+import Layout from '../../components/layout'
+import Head from '../../components/Common/Head'
+import CaseStudyHero from '../../components/Common/CaseStudyCards/CaseStudyHero'
+import {
+  TextColumnsBlock,
+  TextAndImageBlock,
+  FullWidthBlock,
+  VideoBlock,
+  ImagesBlock
+} from '../../components/Common/CaseStudyCards/CaseStudyBlocks'
+
+// import SubtitleWithBody from '../../components/Common/SubtitleWithBody'
+// import Statement from '../../components/Common/Statement'
+import CaseStudyPreview from '../../components/Common/CaseStudyCards/CaseStudyPreview'
+
+const BlockRow = styled(Row)`
+  flex-direction: ${({ rowReverse }) => `row${rowReverse ? '-reverse' : null}`};
+  align-items: ${({ alignCenter }) => (alignCenter ? 'center' : null)};
+  padding-top: ${({ theme, mobile }) =>
+    mobile && mobile.top ? theme.space[mobile.top] : null};
+  padding-bottom: ${({ theme, mobile }) =>
+    mobile && mobile.bottom ? theme.space[mobile.bottom] : null};
+  width: ${({ widthPerc }) => widthPerc};
```

Comment:
width should not be here

Reply:
Yep. Will remove. I just didn't take care of it yet.

Reply:
Using the `<Statement>` component solved that issue for us. Resolved.